src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -10161,6 +10186,34 @@ class C
                 Diagnostic(RudeEditKind.NotCapturingVariable, "a", "a"));
         }
 
+        [Fact, WorkItem(51297, "https://github.com/dotnet/roslyn/issues/51297")]
+        public void IndexerWithExpressionBody_Update_LiftedParameter_2()
```

Comment:
>IndexerWithExpressionBody_Update_LiftedParameter_2 [](start = 20, length = 50)

Let's add a variant of this test that uses local function and old-style lambda delegate (unless there is already one somewhere else)